### PR TITLE
fix debian db migrations and service restarts

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
our plugins still call `db:migrate`, so if users have plugins installed, this might still fail…

File: debian/buster/foreman/rules [hunk 1]
```diff
@@ -47,13 +47,9 @@ build:
 	dh $@ --with=systemd
 
 override_dh_systemd_start:
-	dh_systemd_start --no-start --restart-after-upgrade --name=foreman.service --package=foreman-service
-	dh_systemd_start --no-start --restart-after-upgrade --name=foreman.socket --package=foreman-service
-	dh_systemd_start --no-start --name=dynflow-sidekiq@orchestrator --package=foreman-dynflow-sidekiq
-	dh_systemd_start --no-start --name=dynflow-sidekiq@worker --package=foreman-dynflow-sidekiq
+	dh_systemd_start --no-start --restart-after-upgrade --package=foreman-service
+	dh_systemd_start --no-start --restart-after-upgrade --package=foreman-dynflow-sidekiq
```
Comment (evgeni) [inline debian/buster/foreman/rules:51]: this is technically a noop, as the package doesn't contain any restartable services (the template isn't restartable), so we *could* drop it…

**Review by ehelms — APPROVED** [review]
Seems sane and brings closer alignment with RPM environments. I can take this for a test drive today or we can merge and see what those pipes say.

**Comment by evgeni** [comment]
let's go for a pipe ;)

**Pull request merged**